doc/recipe/preprocessor.rst

```diff
@@ -1312,6 +1313,39 @@ use ``coords: [year, day_of_year]``.
 See also :func:`esmvalcore.preprocessor.amplitude`.
 
 
+.. _trend:
+
+Trend
+=====
+
+The ``_trend.py`` module contains the following preprocessor functions:
```

Comment:
```suggestion
The trend module contains the following preprocessor functions:
```
It would be better not to mention private Python modules that only esmvalcore developers need to know about in documentation aimed at the general public (who may not even know Python, because they're developing an R or NCL diagnostic).